.flake8

```diff
@@ -10,7 +10,7 @@ ignore = E203, E501, W503
 per-file-ignores =
     kuma/settings/local.py:F403,F405
     kuma/settings/prod.py:F403,F405
-    kuma/settings/testing.py:F403,F405
+    kuma/settings/pytest.py:F403,F405
```

Comment:
I don't know why but that name makes me nervous. It's called exactly like the [package](https://pypi.org/project/pytest/) and it might just work but it feels doesn't feel solid. How about `pytesting.py` which rings a bit better. You're using these settings when pytest'ing :)